src/patterns/data/default/entity_attribute_part_of.tsv

```diff
@@ -3198,3 +3181,4 @@ OBA:1000939		GO:0030916	otic vesicle formation	PATO:0000001	quality
 OBA:1001091	egg shell trait	UBERON:0005079	eggshell	PATO:0000001	quality
 OBA:2020001	life cycle stage trait	UBERON:0000105	life cycle stage	PATO:0000001	quality
 OBA:2020002	plant structure trait	PO:0009011	plant structure	PATO:0000001	quality
+OBA:1000705		GO:0097009	energy homeostasis	PATO:0000001	quality
```

Comment:
Why is this added here, when all the other ones are removed?

Reply:
I had second thoughts about changing the pattern for this one.
Do we want to use [ENVO:2000015	energy](https://www.ebi.ac.uk/ols/ontologies/envo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FENVO_2000015) as an entity?

Reply:
Sounds wrong to me, but what is meant by the "energy" in "energy homeostasis" - shouldn't this be some role in CHEBI?

Reply:
No, there is no 'energy' in CHEBI roles.
The entity in 'energy homeostasis' should mean [ 'energy' as the physical entity **E** that is measured in joules (J)](https://en.wikipedia.org/wiki/Energy).

E = mc²

The ENVO definition looks like the best entity match here for 'energy' (from Ancient Greek: ἐνέργεια).

Reply:
Ok, Sounds good, will follow your lead!